dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.webBundle.internal-1.0.feature

```diff
@@ -0,0 +1,17 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.webBundle.internal-1.0
+WLP-DisableAllFeatures-OnConflict: false
+visibility = private
+
+-bundles= \
+ com.ibm.ws.eba.wab.integrator
+
+-features=io.openliberty.servlet.api-3.0; ibm.tolerates:="3.1,4.0,5.0,6.0", \
+  io.openliberty.webBundle.internal.ee-6.0; ibm.tolerates:="7.0,8.0,9.0,10.0"
```

Comment:
That is the whole point.  If I had one that covered 6, 7, 8 it would have tolerates for servlet-3.0, 3.1,4.0,5.0,6.0 and then any feature that would depend on this feature would also need to depend on a servlet public feature.  I did it this way so that it wouldn't break existing features that depend on webBundle today.  They will still get servlet-3.0, 3.1, 4.0 and 5.0 like it was before, but they won't get 6.0 public feature because the 10.0 version brings in the private features that only has the web container servlet function, but does not expose the API to the application.

Reply:
Ah, the bit I'd forgotten is that we only have an internal feature for servlet-6.0